mpmp/src/model/Player.java

```diff
@@ -132,6 +135,41 @@ public void useUnjailCard() {
 		unjails--;
 	}
 
+	/**
+	 * Move a number of fields. If you pass the start, you get paid.
+	 */
+	public void move(int distance) {
+		teleport((pos + distance) % Field.Nfields, true);
+	}
+
+	/**
+	 */
+	public void teleport(int pos, boolean passStart) {
```

Comment:
OK, I will do that tomorrow. Makes sense, `move` can fail if you are in prison; same with `teleport`.